patches/minecraft/net/minecraft/client/renderer/ItemRenderer.java.patch

```diff
@@ -1,6 +1,14 @@
 --- a/net/minecraft/client/renderer/ItemRenderer.java
 +++ b/net/minecraft/client/renderer/ItemRenderer.java
-@@ -59,7 +59,7 @@
+@@ -47,6 +47,7 @@
+ import net.minecraft.world.World;
+ import net.minecraftforge.api.distmarker.Dist;
+ import net.minecraftforge.api.distmarker.OnlyIn;
++import net.minecraftforge.client.model.data.IModelData;
```

Comment:
In patches, you don't want to add imports. Instead use the fully-qualified names.

Reply:
Oh okay. Thank you for letting me know. Haven't messed with patches before so it's really appreciated. I'll try to get a fix done today, thanks